src/codeu/chat/client/simplegui/CreateAccountPanel.java

```diff
@@ -0,0 +1,42 @@
+package codeu.chat.client.simplegui;
+
+import javax.swing.*;
+import java.awt.*;
+
+/**
+ * Created by Suveena on 5/18/17.
```

Comment:
While this is ok to keep track..... tells me nothing about what the createaccountpanel class does or where it's used.

You can add, for example, that the class is used for creating a user account and handles all associated server calls? or maybe warn that this class is simply layout and depends on something else for server calls